apps/wavelet/CMakeLists.txt

```diff
@@ -5,10 +5,23 @@
 # Generators in a simple app.
```

Comment:
Why is this CMakeLists.txt so much more complicated than the ones in the other apps? Does multiple generators really mess things up that much?